config/sidekiq_production.yml

```diff
@@ -0,0 +1,18 @@
+SparklineScheduler:               # rake cache_refresh:sparkline
+  cron: '0 0 19 * * * Etc/UTC'
+
+CoinSlugsScheduler:               # rake generate_coin_slugs
+  cron: '0 0 1 * * * Etc/UTC'
+
+SitemapScheduler:                 # rake sitemap:refresh
+  cron: '0 0 0 * * * Etc/UTC'
+
```

Comment:
Hm, might be worth consolidating around a standardized naming and directory layout convention; here's my proposal:

Class names should be descriptive and start with a verb, i.e. `RefreshSparkline`. 

All Scheduler jobs are located in the `lib/scheduled_jobs` directory. As a result, there's no need to append `Scheduler` to the class name.

Lemme know your thoughts on this!

Reply:
Basically the idea behind this standardization is to make it easier to tell at a glance what each scheduler job class does.

For example, if a dev sees `CoinSlugsScheduler` in the `sidekiq_production.yml` file, it's not immediately obvious that the job is to generate coin slugs, whereas `GenerateCoinSlugs` makes it more clear without having to look at the source.

Reply:
Yeah sounds good. I initially put them all there because only the root lib folder was being autoloaded. But I guess it makes more sense to just add another folder to autoloading than to avoid putting them all into a folder.